### PR TITLE
Update background contrast

### PR DESCRIPTION
**Pull request by gregfuruiye**
small tweaks than include appropriate transparency

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ export const hpe = deepFreeze({
         light: '#FFFFFF',
       },
       'background-contrast': {
-        dark: '#FFFFFF12',
-        light: '#11111108',
+        dark: '#1FFFFFFF',
+        light: '#99CCCCCC',
```
Comment (ericsoderberghp) [inline src/js/index.js:42]: This looks like we've swapped the opacity to the first bytes?

Comment (gregfuruiye) [reply src/js/index.js:42]: @ericsoderberghp I was following this convention:  https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4#how-to-set-transparency-with-hex-value-

Comment (ericsoderberghp) [reply src/js/index.js:42]: I believe that's only for Android. Here's the way grommet does it for regular browsers: https://css-tricks.com/8-digit-hex-codes/

Comment (ShimiSun) [reply src/js/index.js:42]: Thanks for catching that @ericsoderberghp

**Pull request merged**